### ISSUE TITLE
"Run Without Debugging" DOES stop at breakpoint 

### ISSUE THREAD

**Issue by idebtor** [microsoft/vscode#113893]
- VSCode Version: 1.52.1
- OS Version: Windows_NT x64 10.018363

Steps to Reproduce:

1. place a breakpoint in c++ code 
2. select "Run Without Debugging", the program stops at the breakpoint. 

Is this as designed or a bug? At least, the visual studio does not work in this way. It is quite embarrassing to  VS users like me. Thank you.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by isidorn** [comment]
I just tried this and it works just fine for me with javascript and latest vscode insiders.
Please file this issue against the c++ extension. Thanks

**Comment by idebtor** [comment]
You have not tested exactly. Please try it with c/c++ code.  I am afraid that it may not be caused by c++ extension, but by Code.

**Comment by gjsjohnmurray** [comment]
@idebtor c++ is not natively supported by VS Code. I don't think it is fair to expect @isidorn to install an extension to investigate your issue unless the developers of the extension first investigate and assert that the problem lies in VS Code.

**Comment by idebtor** [comment]
Thanks for your comments. 
As you know, Gdb and c++ existed long before VS Code ever exists.  However, you may be right.  So, I posted the same question with a concrete example to **StackOverflow**. I have not got a response yet.  For your reference: 

[Stackoverflow link](https://stackoverflow.com/questions/65593283/run-without-debugging-does-stop-at-a-breakpoint-in-vs-code-a-bug-or-a-design)

Thanks in advance.

**Comment by gjsjohnmurray** [comment]
I suggest you use "Report Issue" from the Help menu in VS Code. Set the "File on" dropdown to "An extension". Pick your C++ extension. Add title and steps to reproduce. Use the "Preview on GitHub" button. In the browser page that opens you should see the new issue drafted on the correct repo for the extension. Submit it. Then close the Issue Reporter dialog via its [X] titlebar button.

**Comment by idebtor** [comment]
Wow~
Thanks for your suggestion and kind guide to filing it that I never knew. I will do it soon. 
It is very kind of you.
Many thanks~
(a debtor)<><

**Comment by idebtor** [comment]
As @gjsjohnmurray suggested thankfully, I was able to file it. For your information, 

[github filing at vscode-cpptools](https://github.com/microsoft/vscode-cpptools/issues/6755)

Blessings, (a debtor)<><

**Comment by idebtor** [comment]
Hi @gjsjohnmurray, 
Believe it or not, I've got the answer from the Microsoft team(vscode-cpptools), finally. VS Code is supposed to implement it.  I spent days and days. You know how to help others. Your suggestion and guide really helped me and will help many others as well.
Here is the answer for your reference.

[answer](https://github.com/microsoft/vscode-cpptools/issues/6755)

Duplicate of #5680

The cpptools debuggers (cppvsdbg and cppdbg) do not support run without debugging.
We ignore the noDebug boolean in LaunchRequestArguments . We will just start a debug session.
This will need to be implemented.

See: #5680 (comment)
